constitution.md

```diff
@@ -0,0 +1,201 @@
+
+**6.3** The Vice-Chairperson shall assist The Chairperson in all their duties.
+
+**6.4** The Vice-Chairperson shall assume the duties of The Chairperson (unless otherwise directed by a general meeting of the Society) in the event that The Chairperson is absent or is unable to perform their duties.
+
+**6.5** The Secretary shall deal with all correspondance and records of the Society.
+
+**6.6** The Secretary keeps minutes of all Committee meetings, of all general meetings, of the Annual General Meeting, or any other meetings related to the business of the Society when requested by the Committee.
+
+**6.7** The Secretary shall give notice to all members of the Society's meetings, events, and functions through email.
+
+**6.8** The Secretary shall assist the Public Relations Officer to organise fund raising events and sponsorship where necessary.
+
+**6.9** The Treasurer shall administer the funds of the Society and to maintain accounts and budgets.
+
+**6.10** The Treasurer shall be responsible for the maintenance of accounts subject to the decision of the Committee in compliance with the regulations set down the Clubs & Societies Committee of Dublin City University.
+
+**6.11** The Treasurer shall keep full records of income and expenditure of the Society. The Treasurer shall also keep a record of who authorised various expenditures and shall issue receipts when needed.
+
+**6.12** The Treasurer shall sign all cheques and withdrawal forms for the Society's bank account in conjunction with another bank account signatory member of the Committee.
+
+**6.13** Where the Treasurer is decreed as unavailable, another designated person may sign cheques and withdrawal forms for the Society on consultation with the Clubs & Societies Committee.
+
+6.14 The Public Relations Officer (PRO), along with the Secretary, shall be responsible for the advertising of all functions of the Society through all means possible.
+
+**6.15** The Public Relations Officer (PRO), along with the Secretary, shall deal with external correspondence and maintain record of the contribution of outside bodies have had for the Society and to give notice of all members of these contributions.
+
+**6.16** The Systems Administrators shall be held ultimately responsible for the administration, security, and maintenance of the Society's computer systems. They shall be responsible for the notification of the Committee in regard to any breaches of the rules and regulations of the system. They are hereby authorised to take any administrative action necessary to discharge their duties, so long as such actions are constitutional. Such actions are open to review by the Committee.
+
+**6.17** The Webmaster shall be responsible for the design, implementation, and maintenance of the website. They shall work in conjuction with the other members of The Committee in order ot publicise society details via the web.
```

Comment:
Minor spelling error here: `in order ot publicise ...`

Reply:
Thanks, fixed